packages/lodestar/src/api/impl/beacon/blocks/index.ts

```diff
@@ -109,7 +108,6 @@ export class BeaconBlockApi implements IBeaconBlocksApi {
   }
 
   async publishBlock(signedBlock: phase0.SignedBeaconBlock): Promise<void> {
-    await checkSyncStatus(this.config, this.sync);
```

Comment:
should we still call `notWhileSyncing` (or similar) here?  even if the assumption is that someone has already called `produceBlock` which checks sync status, someone could still theoretically call `publishBlock` and having the sync check there would still be helpful, no?

Reply:
Lighthouse doesn't have this check here, only in duties. It good to not prevent actions unless there's a strong reason to do so, specially publishing a block which is a sensitive operation. I don't any immediate issue with publishing a block even if the chain is syncing.